src/test/test_pivx.cpp

```diff
@@ -84,19 +84,19 @@ TestingSetup::TestingSetup(const std::string& chainName) : BasicTestingSetup(cha
         GetMainSignals().RegisterBackgroundSignalScheduler(scheduler);
 
         // Register EvoNotificationInterface
-        g_connman = std::unique_ptr<CConnman>(new CConnman(0x1337, 0x1337)); // Deterministic randomness for tests.
+        g_connman = MakeUnique<CConnman>(0x1337, 0x1337); // Deterministic randomness for tests.
```

Comment:
can use `std::make_unique` directly instead of `MakeUnique` since we already require c++14